### PR TITLE
fix(FEC-10354): Languages and Settings menus failed to be opened by click

### PR DESCRIPTION
**Pull request by dan-ziv**
### Description of the Changes

Needed to add `plugin-transform-spread` (which transforms spread arrays [...foo]) to `.babelrc`.

### CheckLists

- [ ] changes have been done against master branch, and PR does not conflict
- [ ] new unit / functional tests have been added (whenever applicable)
- [ ] test are passing in local environment
- [ ] Travis tests are passing (or test results are not worse than on master branch :))
- [ ] Docs have been updated

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -12,6 +12,12 @@
     "@babel/plugin-transform-flow-strip-types",
     "@babel/plugin-transform-property-mutators",
     "@babel/plugin-proposal-object-rest-spread",
+    [
+      "@babel/plugin-transform-spread",
+      {
+        "loose": true
```
Comment (Yuvalke) [inline .babelrc:18]: loose is required?

Comment (dan-ziv) [reply .babelrc:18]: yes

Comment (OrenMe) [reply .babelrc:18]: @dan-ziv can you please add explanation why

Comment (dan-ziv) [reply .babelrc:18]: The error is raised from `toChildArray` API of preact.
From this method:
`return EMPTY_ARR.concat.apply([], children.map(toChildArray));`
The plugin doesn't know how to spread empty arrays without loose transpiling.

Comment (OrenMe) [reply .babelrc:18]: @Yuvalke please see above

**Pull request merged**